src/components/side_panel/chart/pie_chart/pie_chart_config_panel.ts

```diff
@@ -0,0 +1,24 @@
+import { createRange } from "../../../../helpers";
+import { LineBarPieConfigPanel } from "../line_bar_pie_panel/config_panel";
+
+export class PieConfigPanel extends LineBarPieConfigPanel {
+  static template = "o-spreadsheet-PieConfigPanel";
+
+  doesDataSetContainNegativeValues() {
+    let isNegative = false;
+    this.props.definition.dataSets.forEach((dataSet) => {
+      const getters = this.env.model.getters;
```

Comment:
I don't think that a double `forEach`is the right call here. you just want to stop at the first occurence of a negative value. a double for Loop with an realy return might be better